app/src/main/java/com/breadwallet/tools/util/Utils.java

```diff
@@ -255,4 +285,47 @@ public static String fetchPartnerKey(Context app, PartnerNames name) {
         AnalyticsManager.logCustomEventWithParams(BRConstants._20200112_ERR,params);
         return "";
     }
+    /// Description: 1713522152
+    public static long tieredOpsFee(Context app, long sendAmount) {
+
+        double sendAmountDouble = new Double(String.valueOf(sendAmount));
+        String usIso = Currency.getInstance(new Locale("en", "US")).getCurrencyCode();
+        CurrencyEntity currency = CurrencyDataSource.getInstance(app).getCurrencyByIso(usIso);
+        double doubleRate = currency.rate;
+        double usdInLTC = sendAmountDouble * doubleRate / 100_000_000.0;
+
+        if (isBetween(usdInLTC, 0.00, 20.00)) {
+            return (long) ((0.20 / doubleRate) * 100_000_000.0);
+        }
+        else if (isBetween(usdInLTC, 20.00, 50.00)) {
+            return (long) ((0.25 / doubleRate) * 100_000_000.0);
+         }
+        else if (isBetween(usdInLTC, 50.00, 100.00)) {
+             return (long) ((1.00 / doubleRate) * 100_000_000.0);
+         }
+        else if (isBetween(usdInLTC, 100.00, 500.00)) {
+             return (long) ((2.00 / doubleRate) * 100_000_000.0);
+        }
+        else if (isBetween(usdInLTC, 500.00, 1000.00)) {
+             return (long) ((2.50 / doubleRate) * 100_000_000.0);
+        }
+        else if ( usdInLTC > 1000.00) {
+             return (long) ((3.00 / doubleRate) * 100_000_000.0);
+        }
+        else {
+             return (long) ((3.00 / doubleRate) * 100_000_000.0);
+        }
+    }
+    private static boolean isBetween(double x, double lower, double upper) {
+        return lower <= x && x <= upper;
+    }
+    public static Set<String> litewalletOpsSet(Context app) {
+        List<String> addressList = Collections.singletonList(Utils.fetchPartnerKey(app, PartnerNames.LITEWALLETOPS));
```

Comment:
what does singletonList do?

Reply:
trying to remember but you can google. Im not a big Java dev...still learning and I used this after research ....;)